### PR TITLE
Tag Hwloc v0.3.0

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

File: Hwloc/versions/0.3.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@osx Homebrew
+BinDeps
+Compat 0.4.11
+julia 0.4-
```
Comment (tkelman) [inline Hwloc/versions/0.3.0/requires]: drop the `-` unless you still need to support pre-releases of 0.4

File: Hwloc/versions/0.3.0/requires [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@osx Homebrew
+BinDeps
+Compat 0.4.11
```
Comment (tkelman) [inline Hwloc/versions/0.3.0/requires]: `@static if is_linux()` needs a newer version than this

File: Hwloc/versions/0.3.0/requires (hunk 1 above)
Comment (eschnett) [inline Hwloc/versions/0.3.0/requires]: How do I update things? I have a new commit in my repo; what now?
- Move the tag or not?
- Do I need a new tag?
- Should I open a new pull request?
- Can Julia 0.4's Pkg do this? (PkgDev in 0.5 currently doesn't let me authorize.)

Comment (tkelman) [inline Hwloc/versions/0.3.0/requires]: For changes just to the requirements, the easiest approach is just modifying this file in-place in this branch of your metadata fork. Then make the corresponding change in the package too, which will get propagated for the next tag.

**Comment by tkelman** [comment]
closing, ref #6009

**Pull request closed without merging**